test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.default.test.ts

```diff
@@ -23,10 +23,10 @@ describe('parse bpmn as json for default sequence flow', () => {
   each([
     ['exclusiveGateway'],
     ['inclusiveGateway'],
+    ['callActivity'],
```

Comment:
:question: why changing the order?

Reply:
I added some scpecific tests for call Activity, and after remove them in order to comme back to the previous tests, but I didn't remember the order 😉